app/migrations/Version20180321121625.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+class Version20180321121625 extends AbstractMigration
+{
+    public function up(Schema $schema)
+    {
+        $this->addSql('CREATE TABLE `rememberme_token` (
+            `series` char(88) UNIQUE PRIMARY KEY NOT NULL,
+            `value` char(88) NOT NULL,
+            `lastUsed` datetime NOT NULL,
```

Comment:
`last_used` no?

Reply:
no, we should keep it in camelCase, it is used here: https://github.com/symfony/symfony/blob/master/src/Symfony/Bridge/Doctrine/Security/RememberMe/DoctrineTokenProvider.php#L84